domainservice/detekt.yml

```diff
@@ -2,3 +2,6 @@ style:
   VarCouldBeVal:
     ignoreLateinitVar: true
     includes: [ "**/test/**" ]
+exceptions:
+  TooGenericExceptionThrown:
+    active: false
```

Comment:
bad idea to ignore rules globally. suppress them only in places where it can be refactored